From f74f640889728aeca86cb81cc90e7cac2066a275 Mon Sep 17 00:00:00 2001
From: Tim Lunn <tim@feathertop.org>
Date: Mon, 29 Dec 2014 09:14:36 +1100
Subject: libtracker-extract: fix crash when running unit tests

media_art_process is not set when running tracker-extract-info tests

https://bugzilla.gnome.org/show_bug.cgi?id=742035

diff --git a/src/libtracker-extract/tracker-extract-info.c b/src/libtracker-extract/tracker-extract-info.c
index cf578db..de2c594 100644
--- a/src/libtracker-extract/tracker-extract-info.c
+++ b/src/libtracker-extract/tracker-extract-info.c
@@ -144,7 +144,8 @@ tracker_extract_info_unref (TrackerExtractInfo *info)
 		g_free (info->where_clause);
 
 #ifdef HAVE_LIBMEDIAART
-		g_object_unref (info->media_art_process);
+		if (info->media_art_process)
+			g_object_unref (info->media_art_process);
 #endif
 
 		g_slice_free (TrackerExtractInfo, info);
-- 
cgit v0.10.1

